test/pipelinefinally_test.go

```diff
@@ -144,6 +144,12 @@ func TestPipelineLevelFinally_OneDAGTaskFailed_InvalidTaskResult_Failure(t *test
 					Type:      "string",
 					StringVal: "$(tasks.dagtask3.status)",
 				},
+			}, {
+				Name: "dagtasks-aggregate-status",
+				Value: v1beta1.ArrayOrString{
+					Type:      "string",
+					StringVal: "$(tasks.status)",
+				},
```

Comment:
please add a similar test for aggregate status in when expressions in finally tasks

we have tests for the individual execution statuses below

https://github.com/tektoncd/pipeline/blob/c15d8e1fee0ac4b4ebbe0d516795ca988e5435ba/test/pipelinefinally_test.go#L216-L235

Reply:
added one more `when` expression to check the aggregate status 👍